specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/monitoringAccounts_API.json

```diff
@@ -0,0 +1,556 @@
+          },
+          "204": {
+            "description": "Monitoring account did not exist"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete monitoring account": {
+            "$ref": "./examples/MonitoringAccountsDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Metrics": {
+      "description": "Information about metrics for the monitoring account",
+      "type": "object",
+      "properties": {
+        "prometheusQueryEndpoint": {
+          "description": "The Prometheus query endpoint for the monitoring account",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "internalId": {
+          "description": "An internal identifier for the metrics container. Only to be used by the system",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "IngestionSettings": {
+      "description": "Settings for data ingestion",
+      "type": "object",
+      "properties": {
+        "dataCollectionRule": {
+          "description": "The Azure resource Id of the default data collection rule for this monitoring account.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "dataCollectionEndpoint": {
+          "description": "The Azure resource Id of the default data collection endpoint for this monitoring account.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "MonitoringAccount": {
+      "description": "Properties of a monitoring account",
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "description": "The immutable ID of the monitoring account. This property is read-only.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "metrics": {
+          "description": "Information about metrics for the monitoring account",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Metrics"
+            }
+          ],
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the monitoring account. Set to Succeeded if everything is healthy.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
```

Comment:
Enum

Reply:
Updated, with the possible values

Reply:
Needs to include Canceled value. Ref: [https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html?q=provisioningstate#rpc005-provisioning-state-semantics-must-be-followed](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html?q=provisioningstate#rpc005-provisioning-state-semantics-must-be-followed).

Reply:
Thanks for pointing this out Chris, added Canceled as a possible state